sync-gateway-config.json

```diff
@@ -97,10 +96,12 @@ function(doc, oldDoc){
     access(doc.owner, "task-list." + doc._id + ".users");
   } else if (getType() == "task") {
     /* Write Access */
+    var owner = doc._deleted ? oldDoc.taskList.owner : doc.taskList.owner;
```

Comment:
I see, I'll update the PR to check for the presence of the dictionary in the Sync Function.